package.json

```diff
@@ -12,11 +12,11 @@
     "build": "next build",
     "start": "next start",
     "lint": "yarn lint-ts ; yarn lint-less ; yarn lint-prettier",
-    "lint-ts": "tslint -c tslint.json 'src/**/*.{ts,tsx}'",
+    "lint-ts": "eslint -c eslint.js 'src/**/*.{ts,tsx}'",
     "lint-less": "stylelint src/**/*.less",
     "lint-prettier": "prettier-check 'src/**/*.{ts,tsx,less}'",
     "lint:fix": "yarn lint-ts:fix && yarn lint-less:fix && yarn lint-prettier:fix",
-    "lint-ts:fix": "tslint --fix -c tslint.json 'src/**/*.{ts,tsx}'",
+    "lint-ts:fix": "eslint --fix -c eslint.js 'src/**/*.{ts,tsx}'",
     "lint-less:fix": "yarn lint-less --fix",
     "lint-prettier:fix": "prettier-check --write 'src/**/*.{ts,tsx,less}'",
```

Comment:
Could be have the same basic command structure as our other repos?
https://github.com/dotkom/glowing-fortnight/blob/master/package.json#L12-L15

Reply:
I agree! I'll make it so!

Reply:
Hm, I'm a bit uncertain on the commands. I see we're using another linter to lint `.less`-files.
I could include it in the basic command and run both linters in one command, since linting is one job in my mind.
However, one could perhaps want to know exactly which linter reports about which errors and thus run them separately.
What do you think, @oleast ?

Reply:
I like the idea of running one command to check the entire thing, but for things like CI we should be able to run the all separately.
I think I like the setup you have here :smile: 

On another note, do we actually need the `prettier-check` dependency, it seems from the other repos that prettier can do that fine by itself?

Reply:
Good spot! Removed `prettier-check`!